coco/check.py

```diff
@@ -377,23 +377,15 @@ async def run(self, result: Result):
             if r:
                 reply.update(r)
 
-        for host, result_ in reply.items():
-            if not result_:
-                if self.state_paths:
+        if self.state_paths:
```

Comment:
What is the difference between self.state_paths and self.state_path?

Reply:
I think it's just an ugly way to keep either a single path or multiple.